### PR TITLE
localize all the error messages

### PR DESCRIPTION
**Pull request by zora-wang**
@sharadkjaiswal 
These files changed are to fix hard code error messge.
Relavent Problem link is here : http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-5662
Thank you ~~

### CODE REVIEW

File: src/Engine/ProtoCore/Properties/Resources.en-US.resx [hunk 1]
```diff
@@ -489,4 +489,364 @@
...
+    <value>"#" expected</value>
+  </data>
+  <data name="ident_expected" xml:space="preserve">
+    <value>ident expected</value>
+  </data>
+  <data name="invalid_Associative_AccessSpecifier" xml:space="preserve">
+    <value>invalid Associative_AccessSpecifier</value>
+  </data>
+  <data name="invalid_Associative_AddOp" xml:space="preserve">
+    <value>invalid_Associative_AddOp</value>
```
Comment (sharadkjaiswal) [inline src/Engine/ProtoCore/Properties/Resources.en-US.resx]: These errors don't make any sense to users. @junmendoza, @lukechurch please review these error messages and provide your comments.

**Comment by sharadkjaiswal** [comment]
I'll let @junmendoza review this.

**Comment by junmendoza** [comment]
@zora-wang For now this is good.
Ive done some investigation on the syntax errors on the parser and it is risky to hard code any error message at this point. 
Pls create a youtrack User story where the description is: "Come up with localizable syntax error strings "

LGTM

**Comment by junmendoza** [comment]
Merging this. Pending task to improve parser message logs

**Pull request merged**